lib/wallaby/driver/log_store.ex

```diff
@@ -8,6 +8,10 @@ defmodule Wallaby.Driver.LogStore do
   @doc """
   Appends logs to a session
   """
+  def append_logs(session, logs) when is_binary(session) and not is_list(logs) do
```

Comment:
Not sure why are we making this change? Seems unrelated to hover

Reply:
It is a bug that I discovered while TDDing this feature. It errors when attempting to remove duplicate logs because it calls `--` with a list and a map instead of two lists.

I have also seen this bug in travis logs.

Reply:
Can you send a link to the travis failure on master branch that this change fixes?

Reply:
https://travis-ci.org/keathley/wallaby/jobs/559192090

```
01:05:02.545 [error] GenServer Wallaby.Driver.LogStore terminating
** (ArgumentError) argument error
    :erlang.--(%{"error" => "invalid session id", "message" => "invalid session id", "stacktrace" => "#0 0x55ee2df547a9 <unknown>\n"}, [%{"level" => "SEVERE", "message" => "http://localhost:41715/images/button-image.png - Failed to load resource: the server responded with a status of 404 (Object Not Found)", "source" => "network", "timestamp" => 1563239096742}, %{"level" => "SEVERE", "message" => "http://localhost:41715/images/button-image.png - Failed to load resource: the server responded with a status of 404 (Object Not Found)", "source" => "network", "timestamp" => 1563239096781}])
```

Reply:
I looked through it and the patch makes sense, although I think this could be a separate PR for easier review ;)